packages/neutron-sdk/src/bindings/msg.rs

```diff
@@ -110,6 +110,8 @@ pub enum NeutronMsg {
         timeout_timestamp: u64,
         fee: IbcFee,
     },
+    /// SubmitProposal sends a proposal to neutron's Admin module
```

Comment:
1. Rename to SubmitAdminProposal
2. Write a comment that explicitly states that this type of messages can only be executed by the Neutron DAO